pcdsdevices/epics_motor.py

```diff
@@ -55,109 +44,123 @@ class PCDSMotorBase(FltMvInterface, EpicsMotor):
-    # This attribute changes if the motor is stopped and unable to move 'Stop',
-    # paused and ready to resume on Go 'Paused', and to resume a move 'Go'.
-    motor_spg = Cpt(EpicsSignal, ".SPG")
 
     @property
     def low_limit(self):
         """
-        Returns the lower soft limit for the motor.
-
-        Returns
-        -------
-        low_limit : float
-            The lower soft limit of the motor.
+        The lower soft limit for the motor.
         """
         return self.low_soft_limit.value
 
     @low_limit.setter
     def low_limit(self, value):
-        """
-        Sets the low limit for the motor.
-
-        Returns
-        -------
-        status : StatusObject
-            Status object of the set.
-        """
-        return self.low_soft_limit.put(value)
+        self.low_soft_limit.put(value)
 
     @property
     def high_limit(self):
         """
-        Returns the higher soft limit for the motor.
-
-        Returns
-        -------
-        high_limit : float
-            The higher soft limit of the motor.
+        The higher soft limit for the motor.
         """
         return self.high_soft_limit.value
 
     @high_limit.setter
     def high_limit(self, value):
-        """
-        Sets the high limit for the motor.
-
-        Returns
-        -------
-        status : StatusObject
-            Status object of the set.
-        """
-        return self.high_soft_limit.put(value)
+        self.high_soft_limit.put(value)
 
     @property
     def limits(self):
         """
-        Returns the soft limits of the motor.
-
-        Returns
-        -------
-        limits : tuple
-            Soft limits of the motor.
+        The soft limits of the motor.
         """
         return (self.low_limit, self.high_limit)
 
     @limits.setter
     def limits(self, limits):
-        """
-        Sets the limits for the motor.
-
-        Parameters
-        ----------
-        limits : tuple
-            Desired low and high limits.
-        """
         self.low_limit = limits[0]
         self.high_limit = limits[1]
 
     def enable(self):
         """
-        Sets the motor to enabled.
+        Enables the motor.
 
-        Returns
-        -------
-        status: Status Object
-            Status object of the set
+        When disabled, all EPICS puts to the base record will be dropped.
         """
-        return self.disabled.set(value=0)
+        return self.disabled.put(value=0)
 
     def disable(self):
         """
-        Sets the motor to disabled.
+        Disables the motor.
 
-        Returns
-        -------
-        status: Status Object
-            Status object of the set
+        When disabled, all EPICS puts to the base record will be dropped.
+        """
+        return self.disabled.put(value=1)
+
+    def check_value(self, value):
+        """
+        Check if the motor is disabled
+        Check if the value is within the soft limits of the motor.
```

Comment:
Fix this docstring?

Reply:
I think you mean that it doesn't follow the one-line-summary convention, since the actual text here is accurate. I'll fix it.